packages/web-app-preview/src/App.vue

```diff
@@ -225,15 +225,21 @@ export default defineComponent({
       if (o !== n) {
         this.loadMedium()
       }
+    },
+
+    currentFileContext: {
```

Comment:
Not that easy... the current file context also changes upon navigating through a folder by using the `left` and `right` buttons/arrow-keys, so this watcher would cause some cyclic data updates in that case. Could you just remove the watcher and create a ticket instead? @dschmidt and myself are changing some data loading details in the drive aliases PR. Would make sense wait and solve this issue after the drive aliases PR is merged.